test/common/filter/config_discovery_impl_test.cc

```diff
@@ -110,7 +110,7 @@ config_source: { ads: {} }
     }
 
     return filter_config_provider_manager_->createDynamicFilterConfigProvider(
-        config_source, name, factory_context_, "xds.", last_filter_config, "http");
+        config_source, name, factory_context_, "", last_filter_config, "http");
```

Comment:
Is there a plan to remove the reloadable feature and use `http_filter` as the only stat prefix? I am not sure if it is worth to add a test with `envoy.reloadable_features.top_level_ecds_stats` disabled and stat prefix set.

Reply:
Per https://github.com/envoyproxy/envoy/blob/main/CONTRIBUTING.md#runtime-guarding, we should maintain both code paths for one release cycle. I'll recover coverage for the original path.